tests/sentry/sentry_metrics/test_multiprocess_steps.py

```diff
@@ -286,6 +286,16 @@ def test_process_messages(mock_indexer) -> None:
     compare_message_batches_ignoring_metadata(new_batch, expected_new_batch)
 
 
+try:
+    import pytest_benchmark  # type: ignore  # noqa
+except ImportError:
+    pass
+else:
+
+    def test_process_messages_bench(benchmark):
+        benchmark(test_process_messages)
```

Comment:
Is this intentional (merging the benchmark code) or is it just left over code ?

Reply:
it's intentional but I guess kind of silly to keep dead code in the testsuite. I can remove it if you'd like

Reply:
as long as this does not become part of the test suite, it is ok.